share/completions/climate.fish

```diff
@@ -0,0 +1,82 @@
+# fish completion for climate
```

Comment:
Yeah. While you're at it, ditch the first line. The rest is okay, and I appreciate the link!